src/Rotations.jl

```diff
@@ -29,11 +29,16 @@ export
         RotMatrix,
         SpQuat,
         AngleAxis,
+        RodriguesVec,
         EulerAngles,         # order is in the type specification
         ProperEulerAngles,   # order is in the type specification
 
-        # quaternions as well
+        # Quaternions as well
         Quaternion,
+        nquatrand,
```

Comment:
removed

On 3 June 2016 at 13:12, Chris Foster <notifications@github.com> wrote:

> In src/Rotations.jl
> <https://github.com/FugroRoames/Rotations.jl/pull/3#discussion_r65651785>:
>
> >          Quaternion,
> > +        nquatrand,
>
> Hmm, should this be in the public interface?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/3/files/da78deba58bd4f502ad6fcc11453606a47d9a354#r65651785>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQAxNDkVrZWvKJAA54OHC5w8VCanKlluks5qH5uDgaJpZM4ItIfo>
> .
>